app/v4/createAmendForeignPropertyAnnualSubmission/def1/Def1_CreateAmendForeignPropertyAnnualSubmissionValidator.scala

```diff
@@ -33,6 +34,7 @@ class Def1_CreateAmendForeignPropertyAnnualSubmissionValidator @Inject() (nino:
     extends Validator[CreateAmendForeignPropertyAnnualSubmissionRequestData] {
 
   private lazy val minimumTaxYear = appConfig.minimumTaxV2Foreign
+  private lazy val maximumTaxYear = TaxYear.fromMtd("2024-25")
```

Comment:
Can we add a unit test for all these validator changes to test whether our code returns the right error when the tax year is 2025-26 onwards, please?